xml/ha_storage_protection.xml

```diff
@@ -920,6 +920,14 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
     the quorum and some reasonable assumptions. No &stonith; SBD resource
     primitive is needed in the CIB.
    </para>
+   <important>
+    <title>Do not block <literal>localhost</literal> UDP traffic in the firewall</title>
+    <para>
+     Diskless SBD relies on reformed membership and loss of quorum to achieve fencing.
+     However, Corosync cannot reform a new membership if the local firewall blocks
+     cluster communication. This can cause a split-brain scenario.
+    </para>
+   </important>
```

Comment:
The wording is quite good. 

Only a nitpicking, basically a matter of preference. If I was an user, I probably would like to read more of a piece of advice such as:

Diskless SBD relies on reformed membership and loss of quorum to achieve fencing. Please always make sure the local firewall doesn't block cluster communication. Otherwise, Corosync cannot reform a new membership, which can cause a split-brain scenario that cannot be handled with fencing based on diskless SBD.

What do you think, @tahliar and @epenchev?

Reply:
I'd only take out the "please", otherwise this is great, thanks!

Reply:
I also would skip the "please" :)  and perhaps changed it a bit like:

`Always make sure the local firewall doesn't block cluster communication and localhost UDP traffic.`

I'm not sure whether this looks more technical in depth. 
I think we also need to have something saying that localhost UDP traffic needs not to be blocked, as I think with the current message a cluster administrator might miss this. What go you think @tahliar and @gao-yan ?

Reply:
Ah yes, it's in the title I kind of missed this one, thanks Yan.

Reply:
> It's actually kind of mentioned in the title of the paragraph :-) Anyway AFAIU by "UDP traffic of the node", we do mean "the traffic from/to the current node used for cluster communication"? BTW, unless we are referring to the address of the local loopback device such as "127.0.0.1", we should avoid using the specific term "localhost".

Yes you have a point about the "localhost".
It's the loopback device that usally has "127.0.0.1" that should not be blocked, guess we can just update the title.

Reply:
In that case, we could probably make the title more generic/precise like "Do not block corosync traffic in the local firewall"?

Reply:
> In that case, we could probably make the title more generic/precise like "Do not block corosync traffic in the local firewall"?

Yes that looks better for a title and the description could have an entry for the loopback device.
Something like:

Diskless SBD relies on reformed membership and loss of quorum to achieve fencing.
Corosync traffic should be able pass thru all network interfaces, including the loopback interface and should not be blocked by a local firewall. Otherwise, Corosync cannot reform a new membership, which can cause a split-brain scenario that cannot be handled by diskless SBD fencing.

@tahliar @gao-yan what do you think? and also for removing this line "Please always make sure the local firewall doesn't block cluster communication."

Reply:
Looks good and thorough to me. Thanks, @epenchev !

Reply:
Thanks @gao-yan and @epenchev ! I've updated it with both your suggestions: 

![SDB-new](https://user-images.githubusercontent.com/3069029/177227073-79fb787b-ae5f-41f8-b9d0-c959aa7e7807.png)